tools/make-release-packages.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# This script builds and packages a Vitess release suitable for creating a new
+# release on https://github.com/vitessio/vitess/releases.
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+
+# sudo gem install --no-ri --no-rdoc fpm
+
+source build.env
```

Comment:
FWIW, I first tried running this script from the `tools` directory and hit this snag:
`./make-release-packages.sh: line 11: build.env: No such file or directory`

Reply:
Yeah, the other tools will have the same problem. They are expected to be run from `VTROOT`. In general, the supported ones are make targets which execute from VTROOT.